docs/intro.asciidoc

```diff
@@ -1,169 +1,10 @@
 [[intro]]
 == Introduction
 
-Centralized logging for Java applications with the Elastic stack made easy.
+ECS loggers are formatter/encoder plugins for your favorite logging libraries.
+They make it easy to format your logs into ECS-compatible JSON.
 
+TIP: Want to learn more about ECS, ECS logging, and other available language plugins?
+See LINK_TO_ECS_LOGGING_DOCS.
```

Comment:
I'll update this in an immediate follow-up PR. I can't link to this new book until the corresponding docs PR has been merged

Reply:
Nevermind, was able to change merge order and get this fix into this PR